### ISSUE TITLE
Set currency for import

### ISSUE THREAD

**Issue by colindean** [egh/ledger-autosync#52]
I've got a QFX file from Vanguard. It's denoted in USD, but ledger-autosync outputs transactions with a `$` for USD amounts. I'd prefer if I could have it output USD to conform to the rest of my records. I don't see a way to do that presently.

**Comment by egh** [comment]
Seems like a good idea. I prefer `$`but I guess that is a personal choice :)

**Comment by colindean** [comment]
Adding a commodity directive worked for me:

```
commodity USD
  alias $
```

This was previously undocumented because now been added to to the ledger documentation as of last night.

**Comment by 501st-alpha1** [comment]
> Adding a commodity directive worked for me:

Do you mean that ledger-autosync now outputs transactions with `USD` instead of `$`?  I was trying to do the inverse in #46, but it didn't work for me when importing a CSV file.

**Comment by colindean** [comment]
No, setting that directive makes USD and `$` show up as USD in reports.
